spring-kafka/src/main/java/org/springframework/kafka/listener/adapter/AdapterUtils.java

```diff
@@ -124,4 +125,13 @@ public static boolean isCompletableFuture(Class<?> resultType) {
 		return CompletableFuture.class.isAssignableFrom(resultType);
 	}
 
+	/**
+	 * Return the true when type is {@code Continuation}.
+	 * @param  parameterType {@code MethodParameter} parameter type.
+	 * @return type is {@code Continuation}.
+	 * @since 3.2
```

Comment:
All good, but this version has been released already. 
Therefore:
```suggestion
	 * @since 3.2.1
```